src/quality_result_gui/quality_errors_tree_model.py

```diff
@@ -786,3 +786,29 @@ def accept_row(
         quality_error = cast(QualityError, tree_item_value)
 
         return quality_error.geometry.intersects(self._extent)
+
+
+class FilterByShowUserProcessedProxyModel(AbstractFilterProxyModel):
+    def __init__(self, parent: Optional[QObject] = None) -> None:
+        super().__init__(parent)
+
+        self._show_processed_errors: bool = True
+
+    def set_show_processed_errors(self, show_processed_errors: bool) -> None:
+        self._show_processed_errors = show_processed_errors
+        self.invalidateFilter()
+
+    def set_enabled(self, enabled: bool) -> None:
+        if not enabled:
+            self.set_show_processed_errors(False)
+        else:
+            self.set_show_processed_errors(enabled)
```

Comment:
This can be simplified to just:

```python
def set_enabled(self, enabled: bool) -> None:
    self.set_show_processed_errors(enabled)
```
But actually this function is not needed at all as checkbox toggle could just be connected directly to `set_show_processed_errors` in QualityResultManager?

Reply:
Ah yes. This function is obsolent now when we always have boolean value declared